### PR TITLE
Updated Sort command to sort by name, major, yearOfStudy, matricNumber

### PR DESCRIPTION
**Pull request by joel0111**

### CODE REVIEW

**Comment by melkwx** [comment]
Is there a need to sort by matric number?

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -150,7 +150,7 @@ A member can have any number of tags (including 0)
 Examples:
 
 * `memberAdd n/John Doe mn/A0123456Z p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 g/Male
-y/Year 2 m/Chemistry t/Swimming`
+ y/Year 2 m/Chemistry t/Swimming`
```
Comment (melkwx) [inline docs/UserGuide.adoc:153]: Is double space necessary?

**Comment by joel0111** [comment]
Yup when keying in to the CCA points system, the user may want to sort by matricnumber to make it more systematic

File: docs/UserGuide.adoc (hunk 1 above)
Comment (joel0111) [reply docs/UserGuide.adoc:153]: not exactly a double space, the newline is showed as a space, but the output only has 1 space

**Comment by A0143487X-Kevin** [comment]
So its always sorted by matric then matric?

**Comment by joel0111** [comment]
The default is name, and for other variables it will be sorted e.g. By year of study first, then those with in the same year, will be sorted by name

**Pull request merged**